### PR TITLE
Clarify additional WebSocket information.

### PR DESCRIPTION
**Pull request by jeffpk62**

### CODE REVIEW

File: websockets.md [hunk 1]
```diff
@@ -87,7 +87,8 @@ A WebSocket client calls this method with the following query parameters to esta
       Once you establish a connection, you can keep it alive indefinitely.
       You remain authenticated for as long as you keep the connection open.
       You do not need to refresh the access token for an active connection
-      that lasts beyond the token's expiration time.
+      that lasts beyond the token's expiration time. In fact, a connection
```
Comment (scot-taylor-ibm) [inline websockets.md]: I'm waffling on "In fact." Does the sentence lose anything by losing that clause? No big; your call.

File: websockets.md [hunk 2]
```diff
@@ -205,6 +206,8 @@ websocket.onerror = function(evt) { onError(evt) };
 ```
 {: codeblock}
 
+The client can open multiple concurrent WebSocket connections to the service. The number of concurrent connections is limited only by the capacity of the service, which should pose no problems for most users.
```
Comment (scot-taylor-ibm) [inline websockets.md]: "which generally do not pose problems for users"? (conditional!)

File: websockets.md [hunk 3]
```diff
@@ -345,12 +348,16 @@ websocket.send(JSON.stringify(message));
...
+The service terminates the session and closes the connection if an inactivity or session timeout occurs:
+
+-   An *inactivity timeout* occurs if audio is being sent by the client but the service detects no speech. The inactivity timeout is 30 seconds by default. You can use the `inactivity_timeout` parameter to specify a different value, including `-1` to set the timeout to infinity. For more information, see [Inactivity timeout](/docs/services/speech-to-text?topic=speech-to-text-input#timeouts-inactivity).
+-   A *session timeout* occurs if the service receives no data from the client or sends no interim results for 30 seconds. You cannot change the length of this timeout, but you can extend the session by sending the service any audio data, including just silence, before the timeout occurs. You must also set the `inactivity_timeout` to `-1`. You are charged for the duration of any data that you send to the service, including the silence that you send to extend a session. For more information, see [Session timeout](/docs/services/speech-to-text?topic=speech-to-text-input#timeouts-session).
```
Comment (scot-taylor-ibm) [inline websockets.md]: "And you are charged--for silence--" (never mind)

File: websockets.md [hunk 4]
```diff
@@ -345,12 +348,16 @@ websocket.send(JSON.stringify(message));
...
+
+WebSocket clients and servers can also exchange *ping-pong frames* to avoid read timeouts by periodically exchanging small amounts of data. Many WebSocket stacks exchange ping-pong frames, but some do not. To determine whether your implementation uses ping-pong frames, check its list of features. You cannot programmatically determine or manage ping-pong frames.
```
Comment (scot-taylor-ibm) [inline websockets.md]: Is "ping-pong frames" an accepted technical term? Just asking.

File: websockets.md [hunk 5]
```diff
@@ -345,12 +348,16 @@ websocket.send(JSON.stringify(message));
...
 
--   An *inactivity timeout* occurs if audio is being sent by the client but the service detects no speech. The inactivity timeout is 30 seconds by default. You can use the `inactivity_timeout` parameter to specify a different value, including `-1` to set the timeout to infinity.
--   A *session timeout* occurs if the service receives no data from the client or sends no interim results for 30 seconds. You cannot change the length of this timeout, but you can extend the session by sending the service any audio data, including just silence, before the timeout occurs. You must also set the `inactivity_timeout` to `-1`. You are charged for the duration of any data that you send to the service, including the silence that you send to extend a session.
+If your WebSocket stack does not implement ping-pong frames and your are sending long audio files, your connection can experience a read timeout. To avoid such timeouts, continuously stream audio to the service or request interim results from the service. Either approach can ensure that the lack of ping-pong frames does not cause your connection to close.
```
Comment (scot-taylor-ibm) [inline websockets.md]: "Either approach *can* ensure" or "Either approach *ensures*"?

File: websockets.md (hunk 3 above)
Comment (jeffpk62) [reply websockets.md]: Yes, isn't it great!

File: websockets.md (hunk 1 above)
Comment (jeffpk62) [reply websockets.md]: I know it's gratuitous, but I really liked the "in fact" clause.

File: websockets.md (hunk 2 above)
Comment (jeffpk62) [reply websockets.md]: AAAHHHH!  Next thing you know....

File: websockets.md (hunk 1 above)
Comment (scot-taylor-ibm) [reply websockets.md]: Good by me.

File: websockets.md (hunk 3 above)
Comment (scot-taylor-ibm) [reply websockets.md]: In the naked logs I saw
Ten thousand sessions, maybe more...

File: websockets.md (hunk 5 above)
Comment (scot-taylor-ibm) [reply websockets.md]: Per JPK, this is a properly nuanced statement.

**Pull request merged**